### ISSUE TITLE
blockly is not able to load after  blockly_compressed.js file  is updated recently

### ISSUE THREAD

**Issue by GAUTAMRAJU15** [google/blockly#3254]
The blocks suddenly stopped loading in our site after the blocks_compressed file is updated. Can you tell me what exactly is being updated in that file ?

**Comment by NeilFraser** [comment]
A huge amount got updated:
https://github.com/google/blockly/pull/3165

Check your browser's console.  There's probably an error.

If your site is visible online and you post the URL here, we can take a peek.

Thanks!

**Comment by GAUTAMRAJU15** [comment]
actually for now we have reverted back to v2 but  i did checked the console while running the site , there was no error at all, we will try to figure that out later,  we have other priorities as company , i will catch you up in some days about this issue,

**Comment by NeilFraser** [comment]
We won't close the issue.  We are very interested in what compatibility problems this latest release has for users.  Since you are aren't hacking the core, they should be easy to fix.

**Comment by GAUTAMRAJU15** [comment]
it's been a while , so the problem i found is when i do : 
   ``` Blockly.BlockSvg.START_HAT = true;
    Blockly.BlockSvg.START_HAT_HEIGHT = 16;``` 
it gives a hat to each and every block in the workspace, while in v2 it didn't , what do you think about this ?

**Comment by GAUTAMRAJU15** [comment]
in v2 , earlier this line that's globally set only allowed the top level block to get a  hat 
```
   Blockly.BlockSvg.START_HAT = true;
   Blockly.BlockSvg.START_HAT_HEIGHT = 16;
```
but now in v3, each block in the workspace gets a hat , is this an issue or it's meant to be that way since the v1 ?

**Comment by BeksOmega** [comment]
That problem actually got fixed in the [3.20191014.2](https://github.com/google/blockly/releases/tag/3.20191014.2) patch release. The original issue is [here](https://github.com/google/blockly/pull/3280).

So if you upgrade to the latest patch release 3.20191014.4, you should be good!
